### ISSUE TITLE
Headings should be sentence case, not titlecase

### ISSUE THREAD

**Issue by domenic** [w3c/paint-timing#14]
Right now they are inconsistent. W3C standard style is sentence case.

**Comment by thomas-darling** [comment]
Yes, this really should be consistent - and can we please have it changed ASAP.

Pointless inconsistencies like this are incredibly frustrating to have in a standard, especially when they are ignored for so long that they start getting out into the wild - case in point, this just shipped in Chrome 60.

https://www.chromestatus.com/feature/5688621814251520

**Comment by domenic** [comment]
This is just an editorial issue about the casing of headings (e.g. "Processing Model" should be "Processing model"). It doesn't impact what ships, and doesn't become more urgent because Chrome shipped.

**Comment by thomas-darling** [comment]
Wow, sorry about that - somehow I thought this was about something completely different.

What I'm referring to is the casing of the actual entry names `first-paint` and `first-contentful-paint` - their casing is inconsistent with the `Navigation Timing API`, which is rather confusing.
I'll create a separate issue for that one :-)

**Comment by npm1** [comment]
I've updated a couple in https://github.com/w3c/paint-timing/pull/21

Not sure how I feel about "Paint timing" vs "Paint Timing" because it could be considered a proper noun. If you're ok with the latter, then I think we can close this issue. The headings after Acknowledgements are automatically generated and the inconsistency between sentence case and title case seems to be a bikeshed bug.

**Comment by domenic** [comment]
Personally I don't think Paint Timing is a proper noun. Whose name is it?

**Comment by domenic** [comment]
(I guess the misuse of proper nouns is a more general concern, e.g. the terms "first paint", "first contentful paint", are both treated as if they are peoples' names.)

**Comment by npm1** [comment]
Well, "Paint Timing" would be the name of the API. Proper nouns are not restricted to just people names. Similarly, it could be argued that "First Paint" refers to a specific event, etc. It's true, though, that lowercase and uppercase are not used consistently so we should just decide which provides a more clear message and refer to them in a consistent way.

**Comment by domenic** [comment]
Sure, if I were to say "Browser X is shipping the Paint Timing API", then I'd use that. But if I'm talking about the "paint timing" concept, I'd lowercase.

**Comment by npm1** [comment]
All our headings are now sentence case, although there's a couple generated by Bikeshed which are not, so I've filed https://github.com/tabatkins/bikeshed/issues/1609 for that.

**Comment by tabatkins** [comment]
> W3C standard style is sentence case.

Where did you get this information, @domenic? The [W3C Style Guide](https://w3c.github.io/manual-of-style/) says nothing about this, and it itself uses title case mostly consistently.

**Comment by domenic** [comment]
Hmm, I'm not sure why I wrote that. I could have been meaning to what WHATWG standard style, or I could have been under a mistaken impression about the W3C.

My best guess is that I saw they were inconsistent and chose one to push for based on my WHATWG experience. Sorry about that :-/. But I do think https://github.com/tabatkins/bikeshed/issues/1609 would be nice as a configurable option for increasing consistency at least in WHATWG specs, if nothing else.
